### PR TITLE
test: add normal/try catch revert testcase

### PR DESCRIPTION
**Pull request by RetricSu**

### CODE REVIEW

**Review by magicalne — APPROVED** [review]
The build failed as just we can expect.

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: polyjuice-tests/src/test_cases/evm-contracts/revert/contracts.sol [hunk 1]
```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.8.4;
+
+contract DebugAAA {
```
[the remark this answers is not in the document]
Comment (RetricSu) [reply polyjuice-tests/src/test_cases/evm-contracts/revert/contracts.sol:3]: any suggestions for the name? contract A / B / C ?

[the remark this answers is not in the document]
Comment (keroro520) [reply polyjuice-tests/src/test_cases/evm-contracts/revert/contracts.sol:3]: We can learn naming from this directory https://github.com/ethereum/tests/tree/develop/GeneralStateTests/stRevertTest

[the remark this answers is not in the document]
Comment (RetricSu) [reply polyjuice-tests/src/test_cases/evm-contracts/revert/contracts.sol]: updated

**Comment by RetricSu** [comment]
commit [f21f29f](https://github.com/godwokenrises/godwoken-polyjuice/pull/183/commits/f21f29f9e5c2711c88dd77a62aeb6638b9eeb4cf) 

- rename test contracts for better maintainbility
- covert try catch revert test in contract constructor

**Review by magicalne — APPROVED** [review]

File: polyjuice-tests/src/test_cases/evm-contracts/revert/CallRevertWithTryCatch.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "_format": "hh-sol-artifact-1",
+  "contractName": "CallRevertWithTryCatch",
+  "sourceName": "contracts/try.sol",
```
Comment (keroro520) [inline polyjuice-tests/src/test_cases/evm-contracts/revert/CallRevertWithTryCatch.json:4]: should we update "sourceName"? and abi description.

**Review by keroro520 — APPROVED** [review]

**Review by Flouse — APPROVED** [review]

**Pull request merged**